centralledger/chart-handler-transfer-prepare/configs/knexfile.js

```diff
@@ -8,6 +8,8 @@ const Config = require('/opt/central-ledger/src/lib/config')
 module.exports = {
     client: 'mysql',
     connection: Config.DATABASE_URI,
+    min: Config.DB_CONNECTION.DB_CONNECTION_POOL_MIN,
```

Comment:
This should be: `min: Config.DB_CONNECTION_POOL_MIN,`

Reply:
Updated as requested.

Reply:
Updated as requested.